index.jade

```diff
@@ -16,6 +16,17 @@ html(lang='en')
       h1 { font-size: 36px; font-weight: 500 }
       h1 a { color: #222; text-decoration: none }
     link(rel='icon', href='[[ assets/images/favicon.ico ]]', type='image/x-icon')
+    link(rel='apple-touch-icon', sizes='512x512', href='[[ assets/images/ios/icon_512_512.png ]]')
+    link(rel='apple-touch-icon', sizes='256x256', href='[[ assets/images/ios/icon_256_256.png ]]')
+    link(rel='apple-touch-icon', sizes='192x192', href='[[ assets/images/ios/icon_192_192.png ]]')
+    link(rel='apple-touch-icon', sizes='180x180', href='[[ assets/images/ios/icon_180_180.png ]]')
+    link(rel='apple-touch-icon', sizes='152x152', href='[[ assets/images/ios/icon_152_152.png ]]')
+    link(rel='apple-touch-icon', sizes='144x144', href='[[ assets/images/ios/icon_144_144.png ]]')
+    link(rel='apple-touch-icon', sizes='120x120', href='[[ assets/images/ios/icon_120_120.png ]]')
+    link(rel='apple-touch-icon', sizes='114x114', href='[[ assets/images/ios/icon_114_114.png ]]')
+    link(rel='apple-touch-icon', sizes='76x76', href='[[ assets/images/ios/icon_76_76.png ]]')
+    link(rel='apple-touch-icon', sizes='72x72', href='[[ assets/images/ios/icon_72_72.png ]]')
+    meta(name='msapplication-config', content='[[ assets/images/win81/browserconfig.xml ]]')
```

Comment:
There's no `/` in the beginning of URL, it seems it might not work for deeplinks. (I mean, when new user go to the deeplink first, without ever visiting homepage.)